app/src/acceptance/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Why is this needed?

Reply:
So we can inspect the HTTPS traffic using Charles or other proxy applications